light-circuits/createRustVerifyingKey.ts

```diff
@@ -1,2 +1,22 @@
-import { createVerifyingkeyRsFile } from "../light-zk.js/src/cli-utils/createRustVerifyingKey";
-createVerifyingkeyRsFile();
+// import { createVerifyingkeyRsFile } from "../cli/src/utils/createRustVerifyingKey";
+
+// @matteo: paths need to be fixed and both the programName + circuitName should be passed in some way
+
+// const vKeyJsonPath = "./build-circuit/verifyingkey.json";
```

Comment:
Why is this getting commented?

Reply:
a) the original import path didn't exist anymore and has been fixed with the correct one
b) the original method call didn't receive in input any args, the new one expects in input six args

this is in somehow a "deadlock" state, that's why has been commented.
let's talk about unclogging this!  😎

Reply:
I see prior to the move from zk.js to cli createVerifyingkeyRsFile() was a wrapper function around the function it is now which took arguments from argv.
We can leave it like this for this pr.
@xonoxitron can you implement this wrapper function in a separate pr?

Reply:
could not figure out the kind of args! there's some example somewhere?

Reply:
light-circuit build was fixed by @vadorovsky if you rebase and accept the current main state it works.

Reply:
cool, done! ✅